scripts/commit-msg-lint.sh

```diff
@@ -3,11 +3,11 @@
 commit_msg_file=$1
 commit_msg=$(cat $commit_msg_file)
 
-regex="^(feat|fix|docs|style|refactor|perf|test|build|ci|chore|revert)\([a-z]+\): .+$"
+regex="^(feat|fix|docs|style|refactor|perf|test|build|ci|chore|revert): .+$"
```

Comment:
Same question here. Why are we changing this?

Reply:
This is changed as a result of removing the scope from commit message. My reply to your comment in the README expands on it.

(Not sure why I can't respond to the celery_settings.py comment, so I'm including it here)
We're changing the code in celery_settings.py because we want to remove all commented out code. The production code was important, so I created a new ENVIRONMENT env variable that we could check to see if it should be run. @esparr made a comment above wondering if we also need specific settings for the staging environment, apart from dev and prod.